node_modules/oae-core/changepic/js/changepic.js

```diff
@@ -36,9 +36,6 @@ define(['jquery', 'oae.core', 'jquery.jcrop', 'jquery.fileupload', 'jquery.ifram
             principalId: null
         };
 
-        // IE9 and below don't support XHR file uploads and we fall back to iframe transport
-        var useIframeTransport = !$.support.xhrFileUpload && !$.support.xhrFormDataFileUpload;
```

Comment:
When there's some code like this one:
```
if (!useIframeTransport) {
                        // Update the progress bar
                        updateProgress((data.loaded / data.total) * 100);
                    }
```
Should I remove the `if` statement line only and keep the code inside its blocks?